packages/core/admin/admin/src/components/AuthenticatedApp/index.js

```diff
@@ -56,7 +55,11 @@ const AuthenticatedApp = () => {
 
   useEffect(() => {
     if (userRoles) {
-      handleGuidedTourVisibility(userRoles, setGuidedTourVisibilityRef.current);
+      const isUserSuperAdmin = userRoles.find(({ code }) => code === 'strapi-super-admin');
+
+      if (isUserSuperAdmin) {
+        setGuidedTourVisibilityRef.current(true);
```

Comment:
The guided tour is visible only for super-admin users ? 🤔

Reply:
yep indeed that's intended!